src/stream/src/executor/top_n_new.rs

```diff
@@ -447,11 +589,11 @@ mod tests {
             *res.as_chunk().unwrap(),
             StreamChunk::from_pretty(
                 "  I I
+                +  7 6
+                -  7 6
+                -  8 5
+                +  8 5
```

Comment:
Why is it different from previous?

Reply:
Previous implementation always emits DELETE events for old result set then emits INSERT events for new result set. While this version emit diff when new row is inserted into or deleted from result set.